src/main.ts

```diff
@@ -41,8 +41,8 @@ export default class CursorLocation extends Plugin {
   }
 
   private updateShowStatus() {
-    const mode: string = this.app.workspace.getLeaf().getViewState()
-      .state?.mode;
+    const mode: string = this.app.workspace.getActiveViewOfType(obsidian.WorkspaceLeaf)

```

Comment:
Hey @bb010g, did you mean to use `this.app.workspace.getActiveViewOfType(View)` here, like you mentioned in https://github.com/spslater/obsidian-cursor-location-plugin/issues/21#issuecomment-1633526889? That fixes the problem for me

 `this.app.workspace.getActiveViewOfType(obsidian.WorkspaceLeaf)` does not compile

Reply:
Sorry for the delay. You're correct; I transferred this from my live edits in `.obsidian` wrong. Thanks for catching it.